metadata-ingestion/src/datahub/sdk/mlmodel.py

```diff
@@ -70,29 +78,23 @@ def __init__(
         tags: Optional[TagsInputType] = None,
         terms: Optional[TermsInputType] = None,
         domain: Optional[DomainInputType] = None,
-        extra_aspects: ExtraAspectsType = None,
         group: Optional[str] = None,
-        training_jobs: Optional[List[str]] = None,
+        training_jobs: Optional[List[DataProcessInstanceUrn]] = None,
```

Comment:
in general, our inputs should be permissive and our return types should be strict

I should've been more clear initially - we should be accepting both strings and urns here. also use Sequence instead of List for input types

Reply:
done